emily/handler/src/api/handlers/withdrawal.rs

```diff
@@ -336,17 +334,14 @@ pub async fn update_withdrawals(
         let api_state = accessors::get_api_state(&context).await?;
         api_state.error_if_reorganizing()?;
 
-        let is_trusted_key = context.settings.trusted_reorg_api_key == api_key;
         // Signers are only allowed to update withdrawals to the accepted state.
-        if !is_trusted_key {
-            let is_unauthorized = body
-                .withdrawals
-                .iter()
-                .any(|withdrawal| withdrawal.status != Status::Accepted);
+        let is_unauthorized = body
```

Comment:
I might rename this variable (here and elsewhere) to `is_allowed` as it fits better with `403 Forbidden`, where `is_unauthorized` would fit better with `401 Unauthorized` (but this error is about resource permissions and not authentication so `Forbidden` is correct here imo).

Reply:
[9c21d75](https://github.com/stacks-sbtc/sbtc/pull/1586/commits/9c21d7536926a7dfbc6b8b34250f62a2e05d6393)